run.ps1

```diff
@@ -31,12 +41,13 @@ function Select-Tag() {
 
 $scannerinfoPath = "C:\DWOS\scannerinfo.ini"
 if ((!$Manual) -and (Test-Path $scannerinfoPath)) {
-    $tags = Get-TagsFromScannerInfo $scannerinfoPath
+    $tags = Get-TagsFromConfigFiles $scannerinfoPath
 }
 if ($null -eq $tags) {
     $system = Select-Tag "System" @("7Series", "3Series", "medit", "chairside")
     $variant = Select-Tag "Variant" @("dwos", "cares")
-    $tags = Get-Tags $system $variant
+    $synergy = Select-Tag "SpecificIntall" @("None", "Synergy")

```

Comment:
I don't know if the user would understand the question "Select your SpecificIntall:"
I think we should rather put something like "optional software" or "additional installation" or something like that